src/tasks.rs

```diff
@@ -185,7 +186,7 @@ impl AsRef<u32> for EnqueuedTask {
 
 #[derive(Debug, Clone, Deserialize)]
 #[serde(rename_all = "camelCase", tag = "status")]
-pub enum Task {
+pub enum Task<Http: HttpClient> {
```

Comment:
compiler error
```
error[E0392]: parameter `Http` is never used
   --> src/tasks.rs:189:15
    |
189 | pub enum Task<Http: HttpClient> {
    |               ^^^^ unused parameter
    |
    = help: consider removing `Http`, referring to it in a field, or using a marker such as `PhantomData`
```

However, Task feels that this parameter is necessary to call Client
https://github.com/meilisearch/meilisearch-rust/blob/8a4f62d49a2433a1db6d1e06426fb27731403ce5/src/tasks.rs#L263

Reply:
Similar compile errors have been reported by others...